test/e2e/canary_test.go

```diff
@@ -159,6 +159,7 @@ func buildCanaryCurlPod(name, namespace, image, host string) *corev1.Pod {
 //
 // Note that this test will mutate the default ingress controller
 func TestCanaryRouteRotationAnnotation(t *testing.T) {
+	t.Skip("canary route annotation not supported")
```

Comment:
I'm curious, does this test fail? Though it's not supported, is it worth still running the tests?

Reply:
The test doesn't fail. It always takes 5 mins to run (by design). I'll follow up and remove it.

Reply:
Removed in https://github.com/openshift/cluster-ingress-operator/compare/e6c218342f61701142146aa85d6ba94758df116b..c7e09b53798c935b75a8dc14a6bbc906f835ee1a.